index.css

```diff
@@ -521,21 +522,13 @@ section ul, section li {
     }
     
     h4 a {
-        font-size: 18px;
-        word-wrap: break-all;
+        word-break: break-all;
```

Comment:
That's an interesting suggestion. Perhaps `hyphens: manual` along with inserting `&shy;` entities along camel case and syntax boundaries would make sense. `hyphens: auto` is a bit too aggressive when breaking up normal words. The problematic titles we're seeing are all code/API names.